molgenis-data-rest/src/main/java/org/molgenis/data/rest/RestController.java

```diff
@@ -798,14 +798,8 @@ private boolean isUser2fa()
 
 	@PostMapping("/logout")
 	@ResponseStatus(OK)
-	public void logout(HttpServletRequest request)
+	public void logout(@TokenParam(required = true) String token, HttpServletRequest request)
```

Comment:
```POST http://localhost:8080/api/v1/logout``` with header ```x-molgenis-token``` <myToken> results in token being ```null```.

Reply:
Not good enough apperantly to make this a @Component.